rcCollectionroutes.js

```diff
@@ -0,0 +1,69 @@
+var express = require('express');
+
+var bodyParser = require('body-parser');
+
+var urlEncoded = bodyParser.urlencoded({ extended: false });
+
+var rcCollection = require('../models/rcCollectionmodel');
+
+var router = express.Router();
+
+app.use(express.static('public'));
```

Comment:
App isn't defined in this file, so this is a reference error that will prevent your server from running. You can use router instead of app to serve static resources, but I would recommend putting it, meaning your use of the express.static middleware, in your main server file, not in an express router.